samples/MvcSample.Web/Startup.cs

```diff
@@ -98,6 +104,8 @@ public void Configure(IApplicationBuilder app)
             {
                 routes.MapRoute("areaRoute", "{area:exists}/{controller}/{action}");
 
+                routes.MapRoute("formatRoute", "{controller}/{action}/{format}");
+
```

Comment:
oh sorry, I was testing it before we had routing part ready...I have removed this now. Or should I just leave it there?